spec.bs

```diff
@@ -2800,6 +2803,143 @@ This specification defines two [=policy-controlled features=] identified by the
 Issue(WICG/turtledove#522): Move from "`*`" to "`self`".
 
 
+# Handling Direct from Seller Signals # {#handling-direct-from-seller-signals}
+
+This section specifies a manner by which signals may be provided to auctions such that the signals
+are only used within their intended auction.
+
+Scripts running on the page may peform a `fetch()` call using the `{adAuctionHeaders: true}`
+option, which causes the value of the `Ad-Auction-Signals` response header on that request to
+be used to provide signals in auctions running on the page whose seller matches the origin of the
+`fetch()` destination.
+
+<div algorithm="fetch adAuctionHeaders patch">
+Modify [[FETCH]]'s [[FETCH#requests]] to add: 
+
+A [=request=] has an associated <dfn for=request>capture ad auction headers boolean</dfn>.
+Unless stated otherwise it is false.
+
+Modify [[FETCH]]'s <a href="https://fetch.spec.whatwg.org/#requestinit">RequestInit</a> dictionary to add
+an adAuctionHeaders attribute:
+
+<pre class=idl>
+partial dictionary RequestInit {
+  boolean adAuctionHeaders;
+};
+</pre>
+
+The following step will be added to the <a href="https://fetch.spec.whatwg.org/#dom-request">new Request(input, init) constructor steps</a>,
+before step "Set this's [=Request/request=] to |request|":
+
+1. If <var ignore=''>init</var>["{{RequestInit/adAuctionHeaders}}"] <a for=map>exists</a>, then set |request|'s
+  [=request/capture ad auction headers boolean=] to it.
+
+The following step will be added to the <a href="https://fetch.spec.whatwg.org/#concept-http-network-or-cache-fetch">HTTP-network-or-cache fetch</a>
+algorithm, before step "Modify |httpRequest|'s [=request/header list=] per HTTP. ...":
+
+1. If [=request=]'s [=request/capture ad auction headers boolean=] is true, then
+  [=header list/set a structured field value=] given «`Sec-Ad-Auction-Fetch`, the [=structured header/boolean=] ?1» in
+  |httpRequest|'s [=header list=].
+
+Modify [[FETCH]]'s [[FETCH#infrastructure]] to add a new section called "Per Traversable Navigable Structures",
+with the following content:
+
+<h3 dfn-type=dfn>Direct from seller signals key</h3>
+A direct from seller signals key is a [=struct=] with the following [=struct/items=]:

```

Comment:
I think it's much better form to make the DFN on the struct itself, and not inside the header like this. Could you update this and the instances below?

Reply:
Done